Applications/Embedding/Tensorflow/embedding.py

```diff
@@ -3,12 +3,12 @@
 #
 # Copyright (C) 2021 Jijoong Moon <jijoong.moon@samsung.com>
 #
-# @file	embedding.py
-# @date	15 March 2021
-# @brief	This is Simple Embedding Layer Training Example
-# @see		https://github.com/nnstreamer/nntrainer
-# @author	Jijoong Moon <jijoong.moon@samsung.com>
-# @bug		No known bugs except for NYI items
+# @file	  embedding.py
+# @date	  15 March 2021
+# @brief  This is Simple Embedding Layer Training Example
+# @see    https://github.com/nnstreamer/nntrainer
+# @author Jijoong Moon <jijoong.moon@samsung.com>
+# @bug    No known bugs except for NYI items
```

Comment:
I wonder why some parts seem weird in GitHub...
<img width="627" alt="Screen Shot 2021-04-27 at 5 12 15 PM" src="https://user-images.githubusercontent.com/33684401/116208544-fea56400-a77b-11eb-9181-9a5701a9f980.png">

Reply:
It looks fine on my github :)
![image](https://user-images.githubusercontent.com/17230883/116215960-19c7a200-a783-11eb-8b91-5acc7ae21d64.png)